### PR TITLE
Rename VECTOR to VectorType

### PR DESCRIPTION
**Pull request by peterrum**
FYI @simonsticko

### CODE REVIEW

File: include/deal.II/non_matching/mesh_classifier.h [hunk 1]
```diff
@@ -114,9 +114,9 @@ namespace NonMatching
      * Vector. The triangulation attached to DoFHandler is the one that will be
      * classified.
      */
-    template <class VECTOR>
+    template <class VectorType>
     MeshClassifier(const DoFHandler<dim> &level_set_dof_handler,
-                   const VECTOR &         level_set);
+                   const VectorType &     level_set);
```
Comment (kronbichler) [inline include/deal.II/non_matching/mesh_classifier.h:120]: While here, can you remove one empty line here (as there are 2 between this declaration and the next one)?

**Review by kronbichler — APPROVED** [review]

File: include/deal.II/non_matching/mesh_classifier.h (hunk 1 above)
Comment (peterrum) [reply include/deal.II/non_matching/mesh_classifier.h:120]: Done!

**Review by kronbichler — APPROVED** [review]
Thanks!

**Pull request merged**